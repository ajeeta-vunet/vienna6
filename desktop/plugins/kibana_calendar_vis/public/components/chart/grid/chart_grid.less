@vis-hover-opacity: 0.8;

rect {
  opacity: 1;

  &:hover {
    opacity: @vis-hover-opacity;
  }
}

circle {
  opacity: 0;

  &:hover {
    opacity: 1;
    stroke-width: 10px;
    stroke-opacity: @vis-hover-opacity;
  }
}