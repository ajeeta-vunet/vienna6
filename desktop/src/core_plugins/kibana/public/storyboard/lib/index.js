export { saveStoryboard } from './save_storyboard';
//export { getAppStateDefaults } from './get_app_state_defaults';
