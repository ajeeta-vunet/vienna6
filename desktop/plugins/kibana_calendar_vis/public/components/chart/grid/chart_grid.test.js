/* eslint-disable */

/*
 * Licensed to Elasticsearch B.V. under one or more contributor
 * license agreements. See the NOTICE file distributed with
 * this work for additional information regarding copyright
 * ownership. Elasticsearch B.V. licenses this file to you under
 * the Apache License, Version 2.0 (the "License"); you may
 * not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *    http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing,
 * software distributed under the License is distributed on an
 * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
 * KIND, either express or implied.  See the License for the
 * specific language governing permissions and limitations
 * under the License.
 */

import React from 'react';
import moment from 'moment';
import { findDOMNode } from 'react-dom';
import { mount } from 'enzyme';
import { ChartGrid } from './chart_grid';
import { Dispatcher } from '../../../lib';
import { CalendarVisConfig } from '../../../lib/calendar_vis_config';
import { defaultParams, monthViewParams, dayViewParams } from '../../../default_settings';
import aggResponse from '../../../__tests__/agg_response.json';
import { truncateUnusable, replicateDate } from '../../../../test/jest/utils';
import { calculateBounds } from 'ui/timefilter/get_time';

describe('ChartGrid - default', () => {

  let visConfig;
  let visData;
  let dispatcher;
  const fakeVis = {
    params: defaultParams,
    API: {
      timeFilter: {
        getBounds: function () {
          return calculateBounds({
            from: moment(1531026000000),
            to: moment(1531026000000 + 35 * 86400000)
          });
        }
      }
    }
  };

  beforeEach(() => {
    visData = truncateUnusable(aggResponse.rows[0]);
    dispatcher = new Dispatcher().addContainer(document.createElement('div'));
  });

  afterEach(() => {
    visConfig = null;
    visData = null;
  });

  it('should render a full year overview chart grid', () => {
    visConfig = new CalendarVisConfig(fakeVis, defaultParams);
    // replicate data
    visData = replicateDate(visData);
    const adjustSize = jest.fn();
    const gridWrapper = mount(
      <ChartGrid
        type={visConfig.get('type')}
        gridConfig={visConfig.get('grid')}
        vislibData={visData}
        axes={visConfig.get('categoryAxes')}
        dispatcher={dispatcher}
        renderComplete={adjustSize}
      />
    );
    const grid = gridWrapper.instance();
    expect(findDOMNode(grid)).toMatchSnapshot();
  });

  it('should render a full month overview chart grid', () => {
    visConfig = new CalendarVisConfig(fakeVis, monthViewParams);
    // replicate data
    visData = replicateDate(visData, 3);
    const adjustSize = jest.fn();
    const gridWrapper = mount(
      <ChartGrid
        type={visConfig.get('type')}
        gridConfig={visConfig.get('grid')}
        vislibData={visData}
        axes={visConfig.get('categoryAxes')}
        dispatcher={dispatcher}
        renderComplete={adjustSize}
      />
    );
    const grid = gridWrapper.instance();
    expect(findDOMNode(grid)).toMatchSnapshot();
  });

  it('should render a full day overview chart grid', () => {
    visConfig = new CalendarVisConfig(fakeVis, dayViewParams);
    // replicate data
    visData = replicateDate(visData, 0);
    const adjustSize = jest.fn();
    const gridWrapper = mount(
      <ChartGrid
        type={visConfig.get('type')}
        gridConfig={visConfig.get('grid')}
        vislibData={visData}
        axes={visConfig.get('categoryAxes')}
        dispatcher={dispatcher}
        renderComplete={adjustSize}
      />
    );
    const grid = gridWrapper.instance();
    expect(findDOMNode(grid)).toMatchSnapshot();
  });

});
