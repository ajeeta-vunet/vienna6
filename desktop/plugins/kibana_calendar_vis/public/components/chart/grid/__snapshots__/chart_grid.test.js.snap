// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChartGrid - default should render a full day overview chart grid 1`] = `
<svg
  id="all-days"
>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="40"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-1-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="80"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-2-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="120"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-3-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="160"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-4-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="200"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-5-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="240"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-6-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="280"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-7-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="320"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-8-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="360"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-9-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="400"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-10-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="440"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-11-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="480"
      y="60"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-12-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="40"
      y="100"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-1-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="80"
      y="100"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-2-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="120"
      y="100"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-3-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="160"
      y="100"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-4-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="200"
      y="100"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-5-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="240"
      y="100"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-6-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="280"
      y="100"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-7-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="320"
      y="100"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-8-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="360"
      y="100"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-9-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="400"
      y="100"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-10-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="440"
      y="100"
    />
  </g>
  <g>
    <rect
      class="hour"
      height="40"
      id="day-07-08-2018-11-00-PM"
      rx="4"
      ry="4"
      width="40"
      x="480"
      y="100"
    />
  </g>
</svg>
`;

exports[`ChartGrid - default should render a full month overview chart grid 1`] = `
<svg
  id="all-days"
>
  <g>
    <rect
      class="day"
      data-day="1"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-01-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="40"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="2"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-02-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="80"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="3"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-03-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="120"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="4"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-04-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="160"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="5"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-05-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="200"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="6"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-06-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="240"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="7"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-07-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="280"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="8"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-08-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="40"
      y="100"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="9"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-09-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="80"
      y="100"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="10"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-10-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="120"
      y="100"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="11"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-11-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="160"
      y="100"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="12"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-12-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="200"
      y="100"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="13"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-13-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="240"
      y="100"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="14"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-14-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="280"
      y="100"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="15"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-15-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="40"
      y="140"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="16"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-16-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="80"
      y="140"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="17"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-17-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="120"
      y="140"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="18"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-18-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="160"
      y="140"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="19"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-19-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="200"
      y="140"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="20"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-20-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="240"
      y="140"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="21"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-21-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="280"
      y="140"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="22"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-22-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="40"
      y="180"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="23"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-23-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="80"
      y="180"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="24"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-24-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="120"
      y="180"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="25"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-25-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="160"
      y="180"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="26"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-26-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="200"
      y="180"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="27"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-27-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="240"
      y="180"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="28"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-28-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="280"
      y="180"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="29"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-29-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="40"
      y="220"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="30"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-30-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="80"
      y="220"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="31"
      data-month="7"
      data-year="2018"
      height="40"
      id="day-07-31-2018-12-00-AM"
      rx="4"
      ry="4"
      width="40"
      x="120"
      y="220"
    />
  </g>
</svg>
`;

exports[`ChartGrid - default should render a full year overview chart grid 1`] = `
<svg
  id="all-days"
>
  <g>
    <rect
      class="day"
      data-day="1"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-01-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="40"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="2"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-02-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="40"
      y="75"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="3"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-03-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="40"
      y="90"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="4"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-04-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="40"
      y="105"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="5"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-05-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="40"
      y="120"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="6"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-06-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="40"
      y="135"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="7"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-07-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="40"
      y="150"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="8"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-08-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="55"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="9"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-09-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="55"
      y="75"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="10"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-10-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="55"
      y="90"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="11"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-11-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="55"
      y="105"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="12"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-12-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="55"
      y="120"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="13"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-13-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="55"
      y="135"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="14"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-14-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="55"
      y="150"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="15"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-15-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="70"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="16"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-16-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="70"
      y="75"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="17"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-17-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="70"
      y="90"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="18"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-18-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="70"
      y="105"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="19"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-19-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="70"
      y="120"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="20"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-20-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="70"
      y="135"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="21"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-21-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="70"
      y="150"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="22"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-22-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="85"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="23"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-23-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="85"
      y="75"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="24"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-24-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="85"
      y="90"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="25"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-25-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="85"
      y="105"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="26"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-26-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="85"
      y="120"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="27"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-27-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="85"
      y="135"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="28"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-28-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="85"
      y="150"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="29"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-29-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="100"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="30"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-30-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="100"
      y="75"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="31"
      data-month="7"
      data-year="2018"
      height="15"
      id="day-07-31-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="100"
      y="90"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="1"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-01-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="122.5"
      y="105"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="2"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-02-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="122.5"
      y="120"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="3"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-03-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="122.5"
      y="135"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="4"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-04-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="122.5"
      y="150"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="5"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-05-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="137.5"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="6"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-06-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="137.5"
      y="75"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="7"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-07-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="137.5"
      y="90"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="8"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-08-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="137.5"
      y="105"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="9"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-09-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="137.5"
      y="120"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="10"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-10-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="137.5"
      y="135"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="11"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-11-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="137.5"
      y="150"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="12"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-12-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="152.5"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="13"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-13-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="152.5"
      y="75"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="14"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-14-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="152.5"
      y="90"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="15"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-15-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="152.5"
      y="105"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="16"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-16-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="152.5"
      y="120"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="17"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-17-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="152.5"
      y="135"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="18"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-18-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="152.5"
      y="150"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="19"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-19-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="167.5"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="20"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-20-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="167.5"
      y="75"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="21"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-21-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="167.5"
      y="90"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="22"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-22-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="167.5"
      y="105"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="23"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-23-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="167.5"
      y="120"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="24"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-24-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="167.5"
      y="135"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="25"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-25-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="167.5"
      y="150"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="26"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-26-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="182.5"
      y="60"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="27"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-27-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="182.5"
      y="75"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="28"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-28-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="182.5"
      y="90"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="29"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-29-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="182.5"
      y="105"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="30"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-30-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="182.5"
      y="120"
    />
  </g>
  <g>
    <rect
      class="day"
      data-day="31"
      data-month="8"
      data-year="2018"
      height="15"
      id="day-08-31-2018-12-00-AM"
      rx="1.5"
      ry="1.5"
      width="15"
      x="182.5"
      y="135"
    />
  </g>
</svg>
`;
